Extension/src/LanguageServer/configurations.ts

```diff
@@ -1253,11 +1256,27 @@ export class CppProperties {
         }
     }
 
+    private trimPathWhitespace(paths: string[] | undefined): string[] | undefined {
+        const trimmedPaths = [];
+        if (paths !== undefined) {
+            for (const value of paths) {
+                const fullPath = this.resolvePath(value);
+                if (fs.existsSync(fullPath.trim())) {
+                    trimmedPaths.push(value.trim());
+                } else {
+                    trimmedPaths.push(value);
+                }
+            }
+            return trimmedPaths;
+        }
```

Comment:
This should return `undefined` if paths === undefined. I'd prefer a check at the start.

```typescript
if (paths === undefined) {
    return undefined;
}
```